patternfly/patternfly-elements
# [Draft] feat(avatar): PF1:1

### What was done
Removed extra PFE functions
Reorganized markup to match PF
Added PF CSS variables

### Testing Instructions
Load up the deploy preview for this branch
Load up [PatternFly v4 Avatar](https://www.patternfly.org/v4/components/avatar/)
Compare and contrast
Read over the [docs](https://deploy-preview-xxxx--patternfly-elements.netlify.app/components/tile/) and see if anything's missing

### Preview


Link(s) to demo page(s) where this element can be viewed:
- [Link](https://deploy-preview-<pr_number>--happy-galileo-ea79c4.netlify.app/examples/) 

### Related issues

- (#100) Description of issue

#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
